src/components/ha-dialog.ts

```diff
@@ -127,6 +127,7 @@ export class HaDialog extends DialogBase {
         border-radius: var(--ha-dialog-border-radius, 28px);
         -webkit-backdrop-filter: var(--ha-dialog-surface-backdrop-filter, none);
         backdrop-filter: var(--ha-dialog-surface-backdrop-filter, none);
+        background: var(--ha-dialog-surface-background, none);
```

Comment:
You should not default this to none, that means if this variable is set (the default) there will be no background.

Reply:
Good point :+1: